packages/terra-framework-site/src/app/ApplicationHeader.jsx

```diff
@@ -28,27 +32,27 @@ class ApplicationHeader extends React.Component {
     const isCompactHeader = (this.props.layoutConfig.size === 'tiny' || this.props.layoutConfig.size === 'small');
 
     const bidiContent = (
-      <Menu.ItemGroup key="site-bidi" isSelectable dir="ltr" size="medium" onChange={this.props.onDirChange}>
-        <Menu.Item id="ltr" text="ltr" key="ltr" isSelected={this.props.dir === 'ltr'} />
-        <Menu.Item id="rtl" text="rtl" key="rtl" isSelected={this.props.dir === 'rtl'} />
-      </Menu.ItemGroup>
+      <Utility.ItemGroup key="site-bidi" isSelectable dir="ltr" size="medium" onChange={this.props.onDirChange}>
+        <Utility.Item id="ltr" text="ltr" key="ltr" isSelected={this.props.dir === 'ltr'} />
+        <Utility.Item id="rtl" text="rtl" key="rtl" isSelected={this.props.dir === 'rtl'} />
+      </Utility.ItemGroup>
     );
 
     const localeContent = (
-      <Menu.Item
+      <Utility.Item
         text={`Locale: ${this.props.locale}`}
         key="locale"
         subMenuItems={[
-          <Menu.ItemGroup isSelectable key="local-options" onChange={this.props.onLocaleChange} >
-            <Menu.Item id="en" text="en" key="en" isSelected={this.props.locale === 'en'} />
-            <Menu.Item id="en-GB" text="en-GB" key="en-GB" isSelected={this.props.locale === 'en-GB'} />
-            <Menu.Item id="en-US" text="en-US" key="en-US" isSelected={this.props.locale === 'en-US'} />
-            <Menu.Item id="de" text="de" key="de" isSelected={this.props.locale === 'de'} />
-            <Menu.Item id="es" text="es" key="es" isSelected={this.props.locale === 'es'} />
-            <Menu.Item id="fr" text="fr" key="fr" isSelected={this.props.locale === 'fr'} />
-            <Menu.Item id="pt" text="pt" key="pt" isSelected={this.props.locale === 'pt'} />
-            <Menu.Item id="fi-FI" text="fi-FI" key="fi-FI" isSelected={this.props.locale === 'fi-FI'} />
-          </Menu.ItemGroup>,
+          <Utility.ItemGroup isSelectable key="local-options" onChange={this.props.onLocaleChange} >
+            <Utility.Item id="en" text="en" key="en" isSelected={this.props.locale === 'en'} />
```

Comment:
Minor:

Can you use an array of locals instead combined with the map function to output all of these.